values.schema.json

```diff
@@ -34,6 +34,21 @@
       "type": ["null", "object"], "additionalProperties": false, "title": "opal server settings",
       "required": ["port", "policyRepoUrl", "pollingInterval", "dataConfigSources", "broadcastPgsql", "uvicornWorkers", "replicas"],
       "properties": {
+        "image": {
+          "type": ["null", "object"], "additionalProperties": false, "title": "opal-server image settings",
+          "required": ["path", "name"],
+          "properties": {
+            "path": {
+              "type": "string", "title": "opal-server path to image", "default": "/authorizon/"
+            },
+            "name": {
+              "type": "string", "title": "opal-server image name", "default": "opal-server"
+            },
+            "tag": {
+              "type": "string", "title": "opal-server image tag", "default": "0.1.8"
```

Comment:
pretty sure you are supposed to take this version from `.Chart.AppVersion` as before.

Reply:
I added this one in order to install previous version if needed.

But on second thought, if one use the chart, it is for the latest version.
I'll remove this tomorrow.